python/lance/util/duckdb_ext.py

```diff
@@ -0,0 +1,68 @@
+import duckdb
+import os
+import pathlib
+import platform
+import urllib.request
+import zipfile
+
+
+def install_duckdb_extension(version='latest', unsigned=True):
```

Comment:
we dont need `unsigned`, which is always true if install `lance`?

Reply:
done